mlflow/genai/label_schemas/__init__.py

```diff
@@ -93,22 +85,13 @@ def get_label_schema(name: str) -> LabelSchema:
     Returns:
         LabelSchema: The label schema.
     """
-    try:
-        from databricks.agents import review_app
-    except ImportError:
-        raise ImportError(_ERROR_MSG) from None
-
-    app = review_app.get_review_app()
-    label_schema = next(
-        (label_schema for label_schema in app.label_schemas if label_schema.name == name),
-        None,
-    )
-    if label_schema is None:
-        raise ValueError(f"Label schema with name `{name}` not found")
-    return LabelSchema._from_databricks_label_schema(label_schema)
+    from mlflow.genai.labeling.stores import _get_labeling_store  # Nested to avoid circular import
+
+    store = _get_labeling_store()
+    return store.get_label_schema(name)
 
 
-def delete_label_schema(name: str) -> "ReviewApp":
+def delete_label_schema(name: str):
```

Comment:
The fact that `delete_label_schema` returns a `ReviewApp` instance is strange. I think we should preserve the behavior on Databricks (backcompat), but it seems more appropriate to return `None` in OSS.

Accordingly, I've omitted a type hint and return type docstring here. Users don't expect deletion to return a ReviewApp object, and None doesn't need to be documented.

This seems like a reasonably safe, backwards-compatible middle ground